book/src/project-policies.md

```diff
@@ -0,0 +1,61 @@
+# Versioning, Support, and Kubernetes Compatibility
+
+## Versioning
+
+Each Kubernetes CSI component version is expressed as *x.y.z*, where *x* is the
+major version, *y* is the minor version, and *z* is the patch verseion,
+following [Semantic Versioning](https://semver.org/).
+
+Patch version releases only contain bug fixes that do not break any backwards
+compatibility.
+
+Minor version releases may contain new functionality that do not
+break any backwards compatibility (except for alpha features).
+
+Major version releases may contain new functionality or fixes that may break
+backwards compatibility with previous major releases. Changes that require a
+major version increase include: removing or changing API/flags//behavior, new
+RBAC requirements that are not opt-in, new Kubernetes minimum version
+requirements.
+
+## Support
+
+The Kubernetes CSI project follows the broader Kubernetes project on support.
+Every minor release branch will be supported with patch releases for 1 year.
+
+### Alpha Features
+
+Alpha features are subject to break or be
+removed across Kubernetes and CSI component releases. There is no guarantee alpha
+features will continue to function if upgrading the Kubernetes cluster or
+upgrading a CSI sidecar or controller.
+
+## Kubernetes Compatibility
+
+Each release of a CSI component has a minimum, maximum and recommended Kubernetes version
+that it is compatible with.
+
+### Minimum Version
+
+Minimum version specifies the lowest Kubernetes version where the component will
+function with the most basic functionality, and no additional features added later.
+Generally, this aligns with the Kubernetes version where that CSI spec version was added.
+
+### Maximum Version
+
+Similarly, the max Kubernetes version generally aligns with when support for
+that CSI spec version was removed or if a particular Kubernetes API or feature
+was deprecated and removed.
```

Comment:
I think we should remove "was deprecated". An API gets deprecated much earlier than the actual removal and therefore sidecars continue to be compatible even after deprecation.

Is "Max version" inclusive or exclusive? From the description, it's exclusive (when a required API is removed, the sidecar no longer works), but this might be worth calling out explicitly to avoid confusion.

Reply:
it's supposed to be inclusive. I fixed the wording.